src/browser-frontend/reducers/pages-tabbar-reducers.js

```diff
@@ -39,10 +39,25 @@ function stopTabLoadedAnimation(state, { payload: { pageId } }) {
   });
 }
 
+function moveTabTo(state, {payload: { pageId, newPosition, currentPosition } }) {
```

Comment:
It doesn't seem necessary to pass in `currentPosition` here, since that can be queried from the `displayOrder`.